src/libwhb/src/log.c

```diff
@@ -57,7 +58,8 @@ WHBLogWrite(const char *str)
 BOOL
 WHBLogPrint(const char *str)
 {
-   char *buf = MEMAllocFromDefaultHeapEx(PRINTF_BUFFER_LENGTH, 4);
+   MEMHeapHandle mem2_handle = MEMGetBaseHeapHandle(MEM_BASE_HEAP_MEM2);
```

Comment:
What's the reasoning behind using MEM2 heap instead of Default heap?

From what I understand the default heap should be the best heap to use for this, and unless a game has customised memory allocation then the default heap is normally the same than the mem2 base heap.

Reply:
I based that decision on [this](https://github.com/decaf-emu/decaf-emu/commit/782e530e3699a07ea185f240865d96bcca3db251).

> unless a game has customised memory allocation then the default heap is normally the same than the mem2 base heap.

Okay, in that case, using the default heap would be a much better idea.